timesTable.js

```diff
@@ -0,0 +1,12 @@
+function timesTable (num) {
+  let table = [];
+  for(i = 1; i <= num; i++) {
+    for(x = 1; x <= 5; x++) {
```

Comment:
remember to use `let` when declaring variables also should be `num` rather than 5